requre/online_replacing.py

```diff
@@ -81,3 +84,38 @@ def _internal(*args, **kwargs):
         return _internal
 
     return decorator_cover
+
+
+def store(what: str, in_module: str):
```

Comment:
also in_module would deserve a better name

Reply:
Som suggestion?

Reply:
I would like to be able to read it as a sentence:

```python
@store("function", in_module="the_module")
```
= `Store function in module the_module`.

Reply:
please document the field first, then we can figure out the name - I can't help if I don't know what it does

Reply:
Done. I thought you know what we are discussing.